apps/cyberstorm-nextjs/middleware.ts

```diff
@@ -24,8 +24,10 @@ export async function middleware(request: NextRequest) {
   const req = request;
   const response = NextResponse.next();
 
-  if (request.nextUrl.pathname === "/communities") {
-    return NextResponse.redirect(new URL("/", request.url));
+  if (req.nextUrl.href.includes("/cyberstorm/_next")) {
```

Comment:
What is this needed for? Asking because it would be cleaner if we could just handle the routing shenanigans in the config files of next.js and nginx without resorting to touching the code.